### PR TITLE
fix: information box title type to support node

### PR DESCRIPTION
**Pull request by pphminions**
## Description

Change the information box title to accept node. added data-test-selector for the information-box 

## Motivation and Background Context

The new publisher dashboard use the information box. The title of the information box should be flexible to support not only string.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshots (if appropriate):
The title should be able to format according to the design. 

Screen Shot 2019-06-04 at 11 46 13 am

## Check-list:

- [x] I have read the [Contributing](CONTRIBUTING.md) document.
- [x] I've thought about and labelled my PR/commit message appropriately.
- [x] If this PR introduces breaking changes I've described the impact and migration path for existing applications.
- [x] CI is green (coverage, linting, tests).
- [x] I have updated the documentation accordingly.
- [x] I've two LGTMs/Approvals.
- [x] I've fixed or replied to all my code-review comments.
- [ ] I've manually tested with a buddy.
- [x] I've squashed my commits into one.

### CODE REVIEW

File: src/components/adslot-ui/InformationBox/index.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const InformationBox = ({ children, icon, title, className }) => (
 InformationBox.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
-  title: PropTypes.string,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
Comment (veegandhi) [inline src/components/adslot-ui/InformationBox/index.jsx]: `Proptypes.node` includes strings

Comment (pphminions) [reply src/components/adslot-ui/InformationBox/index.jsx]: this is done

File: src/components/adslot-ui/InformationBox/index.jsx [hunk 2]
```diff
@@ -4,8 +4,8 @@ import SvgSymbol from 'alexandria/SvgSymbol';
 
 require('./styles.scss');
 
-const InformationBox = ({ children, icon, title, className }) => (
-  <div className={`information-box${className ? ` ${className}` : ''}`}>
+const InformationBox = ({ children, icon, title, className, dts }) => (
+  <div className={`information-box${className ? ` ${className}` : ''}`} data-test-selector={dts}>
```
Comment (veegandhi) [inline src/components/adslot-ui/InformationBox/index.jsx]: are you trying to prefix `information-box` to the custom classname provided by the user? if not then you need a space between `information-box` and the string interpolation.

File: docs/examples/InformationBoxExample.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { InformationBox } from '../../src';
 class InformationBoxExample extends React.PureComponent {
   render() {
     return (
-      <InformationBox title="This is an information" icon="./docs/assets/svg-symbols.svg#cancel">
+      <InformationBox title="This is an information" icon="./docs/assets/svg-symbols.svg#cancel" className="test-box">
```
Comment (veegandhi) [inline docs/examples/InformationBoxExample.jsx]: Is the `className="test-box"` used?

**Pull request merged**